### ISSUE TITLE
failed the signing process for extension Page Eraser

### ISSUE THREAD

**Issue by p0rtia** [Noitidart/Chrome-Store-Foxified#49]
can you check it?
After "Sign and Install" i get the error: "Signing failed. Not logged in AMO" but I'm already  logged in into  AMO-Account.
https://chrome.google.com/webstore/detail/page-eraser/ekofpchjmoalonajopdeegdappocgcmj

**Comment by Noitidart** [comment]
Are you in private mode?

**Comment by p0rtia** [comment]
No. vpn, adblk+, blur, noscript

**Comment by Noitidart** [comment]
That's interesting. This bug I have seen only when in private mode. When you got addons.mozilla.org do you see yourself loged in like this:

![](http://i.imgur.com/tXHknhh.png)

**Comment by p0rtia** [comment]
Thats right. But there is funny thing - when I click "Signing failed. Not logged in AMO. **Open Login in New Tab**" i have this:
![firefox1](https://cloud.githubusercontent.com/assets/23333293/20106317/067c23e2-a5d5-11e6-82ca-17dbe5558989.jpg)
when I click CONTINUE, it redirects to login page, and again like above: click CONTINUE and I'm stuck i the loop
LOL

**Comment by Noitidart** [comment]
Oh shoot haha, thanks for sharing that screenshot, I'll try to reproduce this and get back to you as soon as possible. :)

**Comment by p0rtia** [comment]
ok. I've just killed my local firefox profile and started from scratch. And there is a progress  ;)
**Signing failed, max attempts reached while waiting for review to complete. The review may have failed.Open Validation Report in New Tab**
And report looks like that:
![firefox2](https://cloud.githubusercontent.com/assets/23333293/20111380/441f5c06-a5e8-11e6-8500-62dd192ffb33.jpg)
looks like problem with addon?

**Comment by Noitidart** [comment]
Ah yes those are issues previously brought up with the Mozilla team via bugzilla (links should be somewhere in Foxified Issues). They said they wont support anything but `spanning` for `incognito`. And the jquery old versions are banned. :(

**Comment by tenminute** [comment]
I tried this as well the other day but had the same exact issues as p0rtia. Yes, I saw myself logged in and was stuck in that loop. I'm on version 49 and tried Google Translate. I did get it to temporarily install though.

https://play.google.com/store/apps/details?id=com.google.android.apps.translate&hl=en

**Comment by Noitidart** [comment]
Thank for sharing @tenminute  - were you in a private window? Or in a private session? Or is your history set to "Never remember" as in screenshot below:

![](http://i.imgur.com/ZQp2IuA.png)

**Comment by tenminute** [comment]
No private browsing. But yeah... my history is set "Never remember".

**Comment by Noitidart** [comment]
May you please try turning it off then seeing if it works.

**Comment by tenminute** [comment]
Sweet... got it to work. Thanks...

After it signed, I clicked for it to install but then I got an error saying it couldn't install because it was corrupt. I closed the error and clicked install again and it worked (I don't know why that error happened).

**Comment by Noitidart** [comment]
It must be that the manifest.json is using stuff not yet supported by WebExtensions. Google Chrome manifest.json has a lot more. When you got it to install the 2nd time, does the addon actually work?

**Comment by tenminute** [comment]
Seems to be. I messed around with it a little to see if it does what I need it to do.

**Comment by Ordval** [comment]
> May you please try turning it off then seeing if it works.    (Noitidart)

What exactly is it that we need to turn off... (because my 51.0.1 (32 bits) is set to “Remember history”)
...and does the Browser need to be restarted ? I'm logged in to addons.mozilla.org.  Cookies are allowed.  
I'm at Windows 10, trying to install “FreeMyBrowser” or two others of the same kind .

**Comment by Ordval** [comment]
> May you please try turning it off then seeing if it works.    (Noitidart)

What exactly is it that we need to turn off... (because my 51.0.1 (32 bits) is set to “Remember history”)
...and does the Browser need to be restarted ? I'm logged in to addons.mozilla.org.  Cookies are allowed.  
With “Never remember history” set it's not possible to even begin an install of a Chrome Add-On.
I'm at Windows 10, trying to install “FreeMyBrowser” or two others of the same kind .

**Comment by Noitidart** [comment]
@Ordval by turn off I meant to switch to "Remember history" to allow cookies etc. But you are still seeing the same issue?

**Comment by Ordval** [comment]
Yes.

**Comment by TheShivamUp** [comment]
yes I got the same problem with it .

**Comment by melnation-com** [comment]
Just happened to me right now, was upgrading to 15.0.0.30, but ISP was on and off.. and it kinda off during the validation.. 

I can still install 15.0.0.30 if I just clicked "Install Unsigned" yet to test if it works or not.
